spec/fixtures/spec-repos/trunk/CocoaPods-version.yml.etag

```diff
@@ -0,0 +1 @@
+"38ceffa2727d3e9df40f72da77311837-ssl"
```

Comment:
@igor-makarov last question should the `.etag` files be checked in? I dont think we want them.

Reply:
Decided to delete them

Reply:
Makes sense, they aren't used unless podspec is being updated.